src/modules/eventRegister/personal-info-des/PersonalInfoDes.vue

```diff
@@ -5,23 +5,23 @@
       <div class="flex flex-col w-1/3 mr-4">
         <div>First Name</div>
         <BaseTextInput
-          :value="user.firstName"
+          :value="user && user.firstName"
```

Comment:
I do not really agree of using :value because in BaseTextInput already assigned :value=props.modelValue, so just do :modelValue="someText" better?